### ISSUE TITLE
Invalid timezone error

### ISSUE THREAD

**Issue by navjeetc** [magma-labs/magma-chat#70]
I get a "invalid time zone" error on line `Time.use_zone(current_user.time_zone, &block)` in application controller as the code `calendar.time_zone.split('/').last.gsub('_', ' ')` in sessions controller strips of **America** from **America/New_York.** If i remove `.split('/').last.gsub('_', ' ')` all is good. Is that an appropriate fix and should I create a PR or not?

**Comment by softr8** [comment]
Yes please, according to the [documentation](https://developers.google.com/calendar/api/v3/reference/calendars), google returns the timezone `Formatted as an IANA Time Zone Database name, e.g. "Europe/Zurich"` And [Time.find_zone!](https://api.rubyonrails.org/classes/Time.html#method-c-find_zone-21) already expects this format, maybe we need to fallback to app time zone?  this [line](https://github.com/magma-labs/magma-chat/blob/main/app/controllers/sessions_controller.rb#L12) needs to be adjusted as well, and lastly, we might need to create a presenter to show in the UI a formatted timezone name

**Comment by piscespieces** [comment]
Same issue. I tried with @navjeetc approach by deleting `.split('/').last.gsub('_', ' ')` but kept getting this error.

Then I completely deleted the `set_time_zone` from ApplicationController and I still get this error!

```
Invalid Timezone: Chicago
[app/controllers/application_controller.rb:56:in `set_time_zone'](http://localhost:3000/conversations#)
```

I have run already `docker compose down && docker compose up`

I'm just craking up this codebase for first time, not sure what am I doing wrong here
